Freebsd test failures

The job is green but fails on:

```
   Compiling uu_dir v0.0.14 (/Users/runner/work/coreutils/coreutils/src/uu/dir)
error[E0425]: cannot find value `BACKEND` in this scope
    --> /Users/runner/work/coreutils/coreutils/tests/by-util/test_tail.rs:1467:9
     |
1467 |         BACKEND
     |         ^^^^^^^ not found in this scope
     |
help: consider importing this static
     |
12   | use tail::text::BACKEND;
     |
warning: unused import: `pretty_assertions::assert_ne`
   --> /Users/runner/work/coreutils/coreutils/tests/by-util/test_wc.rs:2:5
    |
2   | use pretty_assertions::assert_ne;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(unused_imports)]` on by default
help: consider adding a `#[cfg(test)]` to the containing module
   --> /Users/runner/work/coreutils/coreutils/target/debug/build/coreutils-ab9f34f2426babb6/out/test_modules.rs:192:1
    |
192 | mod test_wc;
    | ^^^^^^^^^^^^
For more information about this error, try `rustc --explain E0425`.
warning: `coreutils` (test "tests") generated 1 warning
error: could not compile `coreutils` due to previous error; 1 warning emitted
```

--- a/tests/by-util/test_tail.rs
+++ b/tests/by-util/test_tail.rs
@@ -20,11 +20,6 @@ use std::thread::sleep;
 #[cfg(unix)]
 use std::time::Duration;
 
-#[cfg(target_os = "linux")]
-pub static BACKEND: &str = "inotify";
-// #[cfg(all(unix, not(target_os = "linux")))]
-// pub static BACKEND: &str = "kqueue";
-
 static FOOBAR_TXT: &str = "foobar.txt";
 static FOOBAR_2_TXT: &str = "foobar2.txt";
 static FOOBAR_WITH_NULL_TXT: &str = "foobar_with_null.txt";
@@ -1445,6 +1440,8 @@ fn test_retry9() {
     // inspired by: gnu/tests/tail-2/inotify-dir-recreate.sh
     // Ensure that inotify will switch to polling mode if directory
     // of the watched file was removed and recreated.
+
+    use tail::text::BACKEND;
 
     let ts = TestScenario::new(util_name!());
     let at = &ts.fixtures;

--- a/tests/by-util/test_wc.rs
+++ b/tests/by-util/test_wc.rs
@@ -1,6 +1,3 @@
-#[cfg(all(unix, not(target_os = "macos")))]
-use pretty_assertions::assert_ne;
-
 use crate::common::util::*;
 
 // spell-checker:ignore (flags) lwmcL clmwL ; (path) bogusfile emptyfile manyemptylines moby notrailingnewline onelongemptyline onelongword weirdchars
@@ -404,6 +401,7 @@ fn test_read_from_nonexistent_file() {
 #[test]
 #[cfg(any(target_os = "linux", target_os = "android"))]
 fn test_files_from_pseudo_filesystem() {
+    use pretty_assertions::assert_ne;
     let result = new_ucmd!().arg("-c").arg("/proc/cpuinfo").succeeds();
     assert_ne!(result.stdout_str(), "0 /proc/cpuinfo\n");
 }

